### ISSUE TITLE
Scoreboard issue

### ISSUE THREAD

**Issue by dutchy1001** [alkarinv/BattleArena#258]
hi,
I have a conflict between BattleArena and NametagEdit. When a player joins (in this case spleef) it plays fine. His Tab color is normaly blue but aftert a game of spleef it's white. When ever the other player goes offline after the game of spleef and relogs into the server it boots the other players who played spleef.

There is a solution to turn the scoreboard off in the config that resolves the matter, but it's just a pitty not to use the scoreboards.

if more info is needed just pm me on bukkit dev

regards dutchy

**Comment by alkarinv** [comment]
Hey Dutchy, so I don't get what the conflict is.  I only set colors to players by putting them on a team, or if you have TagAPI enabled by setting that. Once they are done I remove them from the team.

As far as I know a player can be a member of multiple teams (like what NameTagAPI does). So that should just put them back... checking

**Comment by alkarinv** [comment]
Nm. I think I have this.

**Comment by alkarinv** [comment]
So here is the update. NametagEdit uses Packet manipulation and not the default Bukkit system to set the names.  What this means is there is no good way of getting or setting the names they have set (because they did it straight through the packets).

I then tried using the NametagEdit API.  This resulted in crashing my client many times over as the API kept throwing NullPointerErrors and Scoreboard manipulation exceptions with the most basic commands ( like player.setPrefix). 

Basically, they don't use Bukkit and there is no way to get the prefix/suffix w/o using their API... and their API crashes clients because they don't perform sufficient checks. So I won't be trying to work around this more than I have unless they fix their API.

Contact the authors and tell them that their API conflicts with any other plugin that uses scoreboards and have them fix that up and I can revisit it.

**Comment by dutchy1001** [comment]
Thx for  looking into this matter. As I already had contact them before about this matter I might have to change to something else then NametagEdit for color tab names.

thx again for the explanation

**Comment by alkarinv** [comment]
Yeah, I don't mind supporting other plugins, most of the time they are easy changes and almost no code.  But in this case they are doing some very dodgy(and dangerous) packet manipulation behind the scenes... No clue why they didn't just use the normal Bukkit API for this.
